out_writeapi.go

```diff
@@ -333,6 +333,11 @@ func getInstanceCount() int {
 	return len(configMap)
 }
 
+// this is a test-only method that provides the current offset of the passed in config struct
+func getOffset(config *outputConfig) int64 {
```

Comment:
Why does this function take a config pointer? Instead, all it needs is the instance ID. Internally it can lookup the ID in the config. The test code itself should not need to know about "configmap".

Reply:
Done.